db/migrate/20230603221336_create_decks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateDecks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :decks do |t|
+      t.references :player, foreign_key: true
+      t.string :side
```

Comment:
for compatibility with the new API, how about the following?

side -> side_id
identity -> identity_card_id

and normalize those values to those in the new API?

Reply:
Done